src/style.css

```diff
@@ -2,0 +1,144 @@
+.stars {
+  margin: 0;
+}
+
+.stars__list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
```

Comment:
Consider removing the `margin: 0;` declaration from `.stars__list` if it's not necessary, as the `.stars` class already zeroes out the margin. This would adhere to the best practice of not declaring the same property more than necessary.